pcdsdevices/spectrometer.py

```diff
@@ -208,3 +208,55 @@ class VonHamos4Crystal(VonHamosFE):
 
     def __init__(self, prefix, *, name, **kwargs):
         super().__init__(prefix, name=name, **kwargs)
+
+
+class Mono(BaseInterface, Device):
+    """
+    L2S-I NEH 2.X Monochromator
+
+    Axilon mechatronic desig with LCLS-II Beckhoff motion architecture.
+
+    Parameters:
+    -----------
+    preifx : str
+        Base PV for the monochromator
+
+    name : str
+        Alias for the device
+    """
+    # UI representation
+    _icon = 'fa.minus-square'
+
+    # Motor components: can read/write positions
+    # mirror pitch, urad
+    m_pi = Cpt(BeckhoffAxis, ':MMS:M_PI', kind='hinted')
```

Comment:
Would you mind changing these from comments to `doc=''` information?
```suggestion
    m_pi = Cpt(BeckhoffAxis, ':MMS:M_PI', kind='normal', doc='mirror pitch [urad]')
```

It'll show up in the documentation this way.

Also, these subdevices shouldn't be "hinted" (a brief comparison on the kinds is here https://confluence.slac.stanford.edu/display/PCDS/How+to+add+a+device+to+pcdsdevices#Howtoaddadevicetopcdsdevices-Choosingtheright%22kind%22)

I think I'd suggest changing almost all of these to "normal".

Reply:
Thanks for sharing the docs on the different component types, that was never clear to me. I added these doc input variables to the components in `spectrometer.Mono` and `mirror.XOffsetMirror`, but now I'm seeing something strange:

When I run the local test suite I'm now getting 4 errors (`test_evr.py`, `test_interface.py`, `test_lens.py`, and `test_mpod.py`), all of which fail with `ImportError: cannot import name 'be_lens_calcs'`.

I then also found I now have the hidden file in my checkout, `pcdsdevices/.nfs24aa81ac87ba001d0000034d`, and if I try to remove it, I get:
`rm: cannot remove ‘pcdsdevices/.nfs24aa81ac87ba001d0000034d’: Device or resource busy`.

Have you seen this sort of thing before? I'm surprised that editing mirror.py and spectrometer.py had an effect on be_lens_calcs

Reply:
The `be_lens_calcs` comes from a library `pcdscalc`. If you test using pcds-3.4.1 and up, it should be included. I think it may not be present in the dev environment at the moment - we'll have to add it.

That `.nfs*` thing looks like a temporary file from the file system (NFS). I have seen it before, but not sure in what cases they are generated. You shouldn't worry about it too much, though.

Reply:
Hmm, I've tried with both pcds-3.4.1 and the dev environment but still getting that issue. Though interestingly I can import `pcdscalc` in a python session with the latest environment.

And yes the `.nfs*` files is now gone.

Reply:
Usually `.nfs*` files are caches for hanging processes. They go away when the associated process dies.

I managed to forget to install `pcdscalc` into `dev_conda`, I'm going to do that now.

Reply:
Sorry I missed the follow up comments on this. Yes I just checked again and the `.nfs*` file is gone and the tests now pass locally again using `dev_conda`, thanks Zach!